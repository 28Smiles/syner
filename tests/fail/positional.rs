use syner::Syner;

#[derive(Syner)]
struct Positional(usize, bool, String);

fn main() {
}