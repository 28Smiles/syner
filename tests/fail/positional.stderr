error: Expected a named field
 --> tests/fail/positional.rs:4:19
  |
4 | struct Positional(usize, bool, String);
  |                   ^^^^^

error: Expected a named field
 --> tests/fail/positional.rs:4:26
  |
4 | struct Positional(usize, bool, String);
  |                          ^^^^

error: Expected a named field
 --> tests/fail/positional.rs:4:32
  |
4 | struct Positional(usize, bool, String);
  |                                ^^^^^^
